readthedocs/organizations/tests/test_forms.py

```diff
@@ -120,6 +120,24 @@ def test_add_duplicate_invite_by_email(self):
 
 class OrganizationSignupTest(OrganizationTestCase):
 
+    def test_create_organization_with_empy_slug(self):
+        data = {
+            'name': '往事',
```

Comment:
I feel like we should be smarter with slugging here -- can we not generate a slug for other non-ASCII chars? I thought we had a solution for this.

Reply:
Same, but looks like we already are using the slugify function we are using from projects, but that only applies to chars close to roman letters (like á -> a), for chinese and japanese there isn't a direct mapping, we could research if there is something like in python https://en.wikipedia.org/wiki/Romanization_of_Japanese